Bender.yml

```diff
@@ -6,6 +6,11 @@ dependencies:
   common_verification: { git: "https://github.com/pulp-platform/common_verification.git", version: 0.2.0 }
 
 sources:
+  - target: func_sim
```

Comment:
I think you are looking for the `simulation` tag, right?  (https://github.com/fabianschuiki/bender).

Reply:
Not all simulations want the behavioral `tc_sram`. What if you have already generated some macros and the manufacturer provides a simulation model? Then it would be better to use the technology specific `tc_sram` instead of the generic one in simulation.

Reply:
Then please additionally check the `rtl` flag.  
> Set this target when working with the Register Transfer Level description of a design. If this target is set, only behavioural and no technology-specific modules should be used.
Let's no invent string concatenations but use a few orthogonal tags.

Reply:
Added `rtl/tc_sram` to bender target `rtl`.